scripts/contractInteraction/loan_tokens.py

```diff
@@ -339,4 +348,146 @@ def readLiquidity():
     
     tokenContract = Contract.from_abi("Token", address=conf.contracts['WRBTC'], abi=TestToken.abi, owner=conf.acct)
     bal = tokenContract.balanceOf(conf.contracts['ConverterUSDT'])
-    print("supply of rBTC on swap", bal/1e18)
\ No newline at end of file
+    print("supply of rBTC on swap", bal/1e18)
+
+def testSwapsExternal(underlyingTokenAddress, collateralTokenAddress, amount):
+    sovryn = Contract.from_abi("sovryn", address=conf.contracts['sovrynProtocol'], abi=interface.ISovrynBrownie.abi, owner=conf.acct)
+    underlyingToken = Contract.from_abi("TestToken", address=underlyingTokenAddress, abi=ERC20.abi, owner=conf.acct)
+
+    receiver = conf.acct
+    tx = underlyingToken.approve(conf.contracts['sovrynProtocol'], amount)
+    tx.info()
+
+    tx = sovryn.swapExternal(
+        underlyingTokenAddress,
+        collateralTokenAddress,
+        receiver,
+        receiver,
+        amount,
+        0,
+        0,
+        b'',
+        {"value": 0, "allow_revert": True})
+    tx.info()
+
+# Notes: This function will do:
+# 1. tradeOpenAndClosingWithoutCollateral (using amountUnderlying that is sent in the arguments)
+# 2. lendToPool (using amountUnderlying that is sent in the arguments)
+# 3. removeFromPool (50% of the lending)
+# 4. tradeOpenAndClosingWithCollateral (using amountCollateral that is sent in the arguments)
+# 5. Test borrow (using amountCollateral that is sent in the arguments)
+# 6. SwapsExternal (using amountUnderlying that is sent in the arguments)
+#
+# WARN:
+# 1. make sure you have 3 times balance of underlyingTokenAddress
+# 2. make sure you have 2 times balance of amountCollateral
+def wrappedIntegrationTest(loanTokenAddress, underlyingTokenAddress, collateralTokenAddress, amountUnderlying, amountCollateral):
```

Comment:
pls add assertions so that it would run as test
or print expected and actual values with FAILURE note in case of failure